### PR TITLE
Lesson04 semantics

### PR DESCRIPTION
**Pull request by siefimov**

### CODE REVIEW

File: homework04-semantics/index.html [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+                  <li><a href="#header-tag">header</a></li>
+                  <li><a href="#footer-tag">footer</a></li>
+                  <li><a href="#aside-tag">aside</a></li>
+                  <li><a href="#nav-tag">nav</a></li>
+                  <li><a href="#section-tag">section</a></li>
+                  <li><a href="#article-tag">article</a></li>
+                  <li><a href="#figure-tag">figure</a></li>
+               </ul>
+            </nav>
+            <form target="search">
```
Comment (marynabudz1) [inline homework04-semantics/index.html:41]: form not in header 
please check out what header shopuld contain

File: homework04-semantics/index.html [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+               <input type="submit" value="search" />
+            </form>
+         </div>
+      </header>
+      <hr />
+
+      <!-- wrapping block for main section -->
+      <div style="display: flex">
+         <!-- Основні Semantic Elements -->
+         <section style="flex: 1 0 80%">
```
Comment (marynabudz1) [inline homework04-semantics/index.html:52]: why you putevrrything firsdt in div and than section 
in you need wrapper better use inly div no need for section

**Review by marynabudz1 — APPROVED** [review]
change comments

File: homework04-semantics/index.html (hunk 1 above)
Comment (siefimov) [reply homework04-semantics/index.html:41]: okay, I will check out about header

File: homework04-semantics/index.html (hunk 2 above)
Comment (siefimov) [reply homework04-semantics/index.html:52]: thanks, I will ammend

**Pull request merged**